src/pages/api/thirdparty/dataSources/thirdPartyKeystone.ts

```diff
@@ -0,0 +1,36 @@
+import { RESTDataSource } from '@apollo/datasource-rest'
+import type { KeyValueCache } from '@apollo/utils.keyvaluecache'
+import { DateTime } from 'luxon'
+import { print } from 'graphql'
+import { GET_PUBLIC_ARTICLES } from '../operations/getPublicArticles'
+
+export const ARTICLE_PUBLIC_TAGS = {
+  CNOTE: 'C-Note',
```

Comment:
Thought: Since the enum value is `CNOTE` that's what I made the tag name before I realized that the tag needs to be `C-Note`. I like the use of an enum to control what values can be requested, and since this is just the first version I don't know that we need to change it. But my wondering is can we make the enum name and value match? I'll bring this up at eng sync too since I think a sync discussion might be good for this.